src/Filesystem/Filesystem.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CloudPatches\Filesystem;
+
+/**
+ * Filesystem utils.
+ *
+ * @codeCoverageIgnore
+ */
+class Filesystem
+{
+    /**
+     * Determine if directory
+     *
+     * @param string $directory
+     * @return bool
+     */
+    public function isDirectory(string $directory): bool
+    {
+        return is_dir($directory);
+    }
+
+    /**
+     * Determine if a file or directory exists
+     *
+     * @param string $path
+     * @return bool
+     */
+    public function exists($path): bool
+    {
+        return file_exists($path);
+    }
+
+    /**
+     * Get the contents of a file
+     *
+     * @param string $path
+     * @return string
+     *
+     * @throws FileNotFoundException
+     */
+    public function get(string $path): string
```

Comment:
I would suggest renaming this method to `getContent` or `getFileContent`. 
It's not obvious what is doing method `get` without reading method description.